Description: Correct pod NAME issue in LedgerSMB/Locale.pm
 Is missing the dash from the NAME entry.
Bug: http://sourceforge.net/tracker/?func=detail&aid=3472607&group_id=175965&atid=1005191
Author: Robert James Clay <jame@rocasa.us>

--- a/LedgerSMB/Locale.pm
+++ b/LedgerSMB/Locale.pm
@@ -1,7 +1,7 @@
 
 =head1 NAME
 
-LedgerSMB::Locale  Locale handling class for LedgerSMB
+LedgerSMB::Locale - Locale handling class for LedgerSMB
 
 =head1 SYNOPSIS
 
